lms/static/js/groups/views/cohorts.js

```diff
@@ -304,7 +304,7 @@
                                 });
                             }
                         }).render();
-                        this.$('#file-upload-form-file').focus();
+                        this.$('#file-upload-form-file');
```

Comment:
They'll be focused in the natural course of tabbing, we don't need to do anything special here.